worlds/AutoSNIClient.py

```diff
@@ -83,3 +83,7 @@ async def game_watcher(self, ctx: SNIContext) -> None:
     async def deathlink_kill_player(self, ctx: SNIContext) -> None:
         """ override this with implementation to kill player """
         pass
+
+    def on_package(self, ctx: SNIContext, cmd: str, args: dict) -> None:
```

Comment:
```suggestion
    def on_package(self, ctx: SNIContext, cmd: str, args: Dict[str, Any]) -> None:
```
(Imports are already there.)

Reply:
Right, forgot about it.